auslib/db.py

```diff
@@ -414,6 +457,10 @@ def _prepareDelete(self, trans, where, changed_by, old_data_version):
         if self.onDelete:
             self.onDelete(self, "DELETE", changed_by, query, trans)
 
+        potential_required_signoffs = self.getPotentialRequiredSignoffs([orig_row], transaction=trans)
+        if potential_required_signoffs:
+            verify_signoffs(potential_required_signoffs, signoffs)
```

Comment:
I put these here for two reasons: 1) because it seemed a bit weird if AUSTable.insert/update/delete signatures were different than subclass methods of the same name. And 2) Because it seemed like the most obvious way to satisfy https://bugzilla.mozilla.org/show_bug.cgi?id=1310232.

I think putting these checks right beside permission checks is a good enough reason to override both of these, though, and it will make it more obvious which tables actual require signoffs. I'll give this a shot - hopefully the signatures differences don't cause any major problems.